src/main/java/com/sumologic/jenkins/jenkinssumologicplugin/sender/TimestampingOutputStream.java

```diff
@@ -30,13 +31,30 @@ public TimestampingOutputStream(OutputStream stream) {
     wrappedStream = stream;
   }
 
+  public static byte[] getTimestampAsByteArray() {
+    String timeStampStr = "[" + DATE_FORMAT.format(new Date()) + "] ";
+    byte[] timestampBytes = timeStampStr.getBytes();
+
+    return timestampBytes;
+  }
+
+  public static boolean shouldPutTimestamp(byte[] bytes, int i) {
```

Comment:
Would describe logic behind it. IF you have whitespace first, we treat it as multiline log line.